qbraid/transpiler/conversions/cirq/cirq_to_braket.py

```
@@ -147,6 +158,26 @@ def _to_braket_instruction(
+    operation: cirq_ops.Operation,
+    qubit_mapping: dict[int, int],
+) -> list[braket.circuits.Instruction]:
+    """Converts a Cirq measurement operation to Braket Measure instruction(s).
+
+    Cirq measurement keys have no Braket equivalent; readout order follows qubit
+    order. Inverted bits (``invert_mask``) are realized as an X before the measure.
+    """
+    gate = operation.gate
+    instructions = []
+    invert_mask = gate.invert_mask + (False,) * (len(operation.qubits) - len(gate.invert_mask))
+    for qubit, inverted in zip(operation.qubits, invert_mask):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)